.github/workflows/build.yml

```diff
@@ -977,3 +977,12 @@ jobs:
       - uses: actions/checkout@v4
       - name: REUSE Compliance Check
         uses: fsfe/reuse-action@v4
+
+  label_pr:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label PR
+        uses: actions/labeler@v5
```

Comment:
How about putting this in another workflow as advertised [in the documentation](https://github.com/actions/labeler?tab=readme-ov-file#create-workflow)? I believe it'd be better because we typically don't need this action to run on `push` events on `dev` for example.